app/views/articles/index.html.erb

```diff
@@ -23,7 +23,7 @@
   <meta name="twitter:card" content="summary_large_image">
   <%= auto_discovery_link_tag(:rss, "https://dev.to/feed", title: "#{ApplicationConfig['COMMUNITY_NAME']} RSS Feed") %>
 <% end %>
-<%= javascript_pack_tag "homePage", defer: true %>
+<%= javascript_pack_tag "homePage", defer: true, "data-no-instant": true %>
```

Comment:
The homepage pack bundle does not to be reevaluated. It only needs to be loaded once.

Reply:
Just need to look into the scenario when you're onboarded as that's the only time it would need to be loaded as well.

Reply:
I've reverted this change because when coming from a few pages back to the homepage, it does not run. I'm not sure why. I can dig a bit later.